lib/websocket.go

```diff
@@ -20,7 +44,16 @@ type WebsocketHandler interface {
 // and serves to maintain the list of connections
 type wsHandler struct {
```

Comment:
I think this can be renamed to `connections`. let's change the comment to `connections maintains the set of active websocket connections & associated connection metadata`